### PR TITLE
Rhoaiserving using

### PR DESCRIPTION
**Pull request by alxlenc**
Materials for:
* https://github.com/RedHatTraining/AI265/pull/94

### CODE REVIEW

File: deploying/rhoaiserving-using/validate_model_servers.ipynb [hunk 1]
```diff
@@ -0,0 +1,114 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip -qq install transformers\n",
```
Comment (jramcast) [inline deploying/rhoaiserving-using/validate_model_servers.ipynb:9]: Pin the transfomers version or we might run into problems with future releases.

```suggestion
    "!pip -qq install transformers==4.46.3\n",
```

Comment (jramcast) [reply deploying/rhoaiserving-using/validate_model_servers.ipynb:9]: Or we can include a requirements file and then run something like `pip install -r requirements.txt`

**Review by jramcast — APPROVED** [review]

File: deploying/rhoaiserving-using/validate_model_servers.ipynb [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+    "diabetes_url = \"https://diabetes-rhoai-multimodel.apps.ocp4.example.com/v2/models/diabetes/infer\"\n",
+    "iris_url = \"https://iris-rhoai-multimodel.apps.ocp4.example.com/v2/models/iris/infer\"\n",
+    "distilbert_url = \"https://distilbert.rhoai-singlemodel.svc.cluster.local/v1/models/distilbert:predict\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "1. Validate that the diabetes model, deployed in a multimodel-server, responds."
```
Comment (alxlenc) [inline deploying/rhoaiserving-using/validate_model_servers.ipynb:28]:
```suggestion
    "1. Validate that the diabetes model, deployed in a multi-model server, responds."
```

File: deploying/rhoaiserving-using/validate_model_servers.ipynb [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+    "response = utils.send_inference_request(diabetes_url, diabetes_request)\n",
+    "print(\"\\nResponse: \\n\")\n",
+    "response.json()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "2. Validate that the iris model, deployed in a multimodel-server, responds."
```
Comment (alxlenc) [inline deploying/rhoaiserving-using/validate_model_servers.ipynb:49]:
```suggestion
    "2. Validate that the iris model, deployed in a multi-model server, responds."
```

File: deploying/rhoaiserving-using/validate_model_servers.ipynb [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+    "print(\"\\nResponse: \\n\")\n",
+    "response.json()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "3. Validate that the DistilBERT model performs sentiment analysis.\n",
+    "   Note that the distilbert URL is using an internal URL to the OpenShift cluster."
```
Comment (sffrench) [inline deploying/rhoaiserving-using/validate_model_servers.ipynb:71]:
```suggestion
    "   Note that the distilbert URL uses an internal URL for the OpenShift cluster."
```

**Review by sffrench — COMMENTED** [review]
Quick review complete.

**Pull request merged**